### PR TITLE
create map

### PR DESCRIPTION
**Pull request by zoebarth**
Add google map to portfolio in about me section. Center map around Google Headquarters for now, and provide absolutely no justification as to why there's a map on the portfolio.

image

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -107,4 +107,12 @@ function handleSubmit(e) {
   $(this).find('input,textarea').val('');
 }
 
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(document.getElementById('map'), {
```
Comment (mecolmg) [inline portfolio/src/main/webapp/script.js:112]: nit: You can replace the `document.getElementById('map')` with `$('#map')[0]` as per https://learn.jquery.com/using-jquery-core/faq/how-do-i-pull-a-native-dom-element-from-a-jquery-object/

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -108,6 +108,19 @@ html {
...
+  border: thin solid black;
+  height: 500px;
+  width: 80%;
+}
+
+#map-section {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin: 30px 0 30px 0;
```
Comment (mecolmg) [inline portfolio/src/main/webapp/style.css:121]: nit: `margin: 30px 0;`

Comment (zoebarth) [reply portfolio/src/main/webapp/style.css:121]: Done.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (zoebarth) [reply portfolio/src/main/webapp/script.js:112]: Done.

**Pull request merged**